src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -274,6 +274,11 @@ interface DetailsFormLeftProps {
   formError: boolean;
 }
 
+const backgroundTextByVenue: { [key: string]: string } = {
+  [VenueTemplate.themecamp]: "Theme Camp",
+  [VenueTemplate.partymap]: "Party Map",
+};
```

Comment:
I believe `Record<string, string>` is equivalent to this type.

Is it possible for there to be any other venue values here? We don't currently have a default set. Might be good to add one just in case?

Reply:
Should this type be resticted to keys in `BACKGROUND_IMG_TEMPLATES` ?